lib/views/home.dart

```diff
@@ -48,3 +49,64 @@ class HomePage extends StatelessWidget {
     );
   }
 }
+
+ _checkConnectivity(context) async {
+    var result = await Connectivity().checkConnectivity();
+    if (result == ConnectivityResult.none){
+      _displayDialog(
+        context,
+          Text(
+              "No Connection",
+              style: TextStyle(
+                  color: Colors.white,
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.bold,
+                  letterSpacing: 2.0,
+                  fontFamily: "fonts/BalsamiqSans-Bold.ttf"
+              )
```

Comment:
Hey man @KingOsborne add a `textAlign: TextAlign.center` to the two `Text()` widgets in the `_dialog()` method to center the texts. Should look better that way.